src/main/java/com/google/cloud/tools/ide/login/GoogleLoginState.java

```diff
@@ -144,17 +160,18 @@ public HttpRequestFactory createRequestFactory() {
    * @throws IOException if something goes wrong while fetching the token
    */
   public String fetchAccessToken() throws IOException {
```

Comment:
I think I'd rather an API that passes in an account.  I could see the Deploy dialog, for example, adding support to specify the account to use _for only that action_.